src/main/java/seedu/duke/Todo.java

```diff
@@ -0,0 +1,20 @@
+package seedu.duke;
+
+/**
+ * A Todo Task.
+ */
+public class Todo extends Task {
+
+    public Todo(int id, String task) {
+        super(id, task);
+    }
+
+    /**
+     * {@inheritDoc}
+     */
+    @Override
+    public String printTask() {
```

Comment:
I think the method name is slightly misleading. Would getTaskStatus be a better name?
```suggestion
    public String getTaskStatus() {
```

Reply:
Good point, thank you for the input!